frontend/pages/admin/IntegrationsPage/cards/MdmSettings/MacOSMdmPage/components/content/ApplePushCertInfo.tsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+
+import { IMdmApple } from "interfaces/mdm";
+
+import { readableDate } from "utilities/helpers";
+
+import Button from "components/buttons/Button";
+
+interface IApplePushCertInfoProps {
+  baseClass: string;
+  appleAPNInfo: IMdmApple;
+  orgName: string;
+  serverUrl: string;
+  onClickRenew: () => void;
+  onClickTurnOff: () => void;
+}
+const ApplePushCertInfo = ({
+  baseClass,
+  appleAPNInfo,
+  orgName,
+  serverUrl,
+  onClickRenew,
+  onClickTurnOff,
+}: IApplePushCertInfoProps) => {
+  return (
+    <>
+      <dl className={`${baseClass}__page-content ${baseClass}__apc-info`}>
+        <div>
+          <dt>Common name (CN)</dt>
+          <dd>{appleAPNInfo.common_name}</dd>
```

Comment:
We have a component called DataSet that will handle displaying sets of data like this. Here it is in storybook https://fleetdm.com/storybook/?path=/docs/components-dataset--docs

Reply:
Thanks! I've bookmarked the storybook link :)